docs/apache-airflow-providers-apache-hdfs/connections.rst

```diff
@@ -38,15 +36,3 @@ Port
 
 Login
     Effective user for HDFS operations (non-Kerberized).
-
-Extra (optional, connection parameters)
-    Specify the extra parameters (as json dictionary) that can be used in HDFS connection. The following
-    parameters out of the standard python parameters are supported:
-
-    * ``autoconfig`` - Default value is bool: False. Use snakebite's automatically configured client. This HDFSHook implementation requires snakebite.
-    * ``hdfs_namenode_principal`` - Specifies the Kerberos principal to use for HDFS.
-
-    The following extra parameters can be used to configure SSL for Web HDFS Hook:
-
-    * ``use_ssl`` - If SSL should be used. By default is set to `false`.
-    * ``verify`` - How to verify SSL. For more information refer to https://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification.
```

Comment:
these two extras are use in the web HDFS Hook
https://github.com/apache/airflow/blob/3193857376bc2c8cd2eb133017be1e8cbcaa8405/airflow/providers/apache/hdfs/hooks/webhdfs.py#L106-L108

Reply:
Ah.. Missed that one will fix it.

Reply:
Fixed.